seatunnel-dist/release-docs/licenses/LICENSE-SparseBitSet.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
For those libs using standard Apache License 2.0, we can simply say they use standard Apache License 2.0 without copying these over and over again

Reply:
Okay. Thanks for the reminder.：）